sample_reponse.json

```diff
@@ -22,24 +22,16 @@
                 {
                     "id": "metro",
                     "descriptor": {
-                        "name": "ABC Metro Rail Limited"
+                        "name": "Kochi Metro Rail Limited"
```

Comment:
@techframewirk can you Kochi Metro Rail Limited and add a generic Example name?

Reply:
Changed